### PR TITLE
[KO] Update GMT, osu! Alumni

### PR DESCRIPTION
**Pull request by dora-0**
Ready for reviews

### CODE REVIEW

File: wiki/People/The_Team/Global_Moderation_Team/ko.md [hunk 1]
```diff
@@ -1,79 +1,118 @@
...
 
 ## GMT가 되는 방법
 
-**Global Moderation Team** 이 되려면 상당히 어렵고 복잡한 과정을 거쳐야 합니다. 제2언어를 구사할 수 있는 사람들이 종종 GMT 구성원이 될 가능성이 있는 사람들로 지목되며, 이후에 팀 구성원으로서의 잠재력에 대해 논의 후 선출됩니다.
+**Global Moderation Team**이 되려면 상당히 어렵고 복잡한 과정을 거쳐야 합니다. 제2언어를 구사할 수 있는 사람들이 종종 GMT 구성원이 될 가능성이 있는 사람들로 지목되며, 이후에 팀 구성원으로서의 잠재력에 대해 논의 후 선출됩니다.
 
 ## 역할과 책임
 
-Global Moderation Team은 채팅 채널과 포럼의 쾌적한 운용을 책임지고 커뮤니티의 대화에 신경을 써야합니다. 만약 여러분이 채팅(사일, 괴롭힘 등)이나 포럼(스팸)에 대해서 (이것에 한정된 것은 아님) 채팅의 중재 및 게임에 관련되어 도움이 필요하다면 이들에게 연락해주세요.
+Global Moderation Team은 채팅 채널과 포럼의 쾌적한 운용을 책임지고 커뮤니티의 대화에 신경을 써야합니다. 만약 여러분이 채팅(사일, 괴롭힘 등)이나 포럼(스팸)에 관한 문제(이것에 한정된 것은 아님), 또는 게임과 관련된 도움이 필요하다면 이들에게 연락해주세요.
```
Comment (Mikeds36) [inline wiki/People/The_Team/Global_Moderation_Team/ko.md]:
```suggestion
Global Moderation Team은 채팅 채널과 포럼의 쾌적한 운용을 책임지고 커뮤니티의 대화에 신경을 써야합니다. 만약 여러분이 채팅(채팅 제한, 괴롭힘 등)이나 포럼(스팸)에 관한 문제(이것에 한정된 것은 아님), 또는 게임과 관련된 도움이 필요하다면 이들에게 연락해주세요.
```

File: wiki/People/The_Team/Global_Moderation_Team/ko.md [hunk 2]
```diff
@@ -1,79 +1,118 @@
...
-| ![][flag_CL] [WalterToro](https://osu.ppy.sh/users/5281416) | 스페인어 | 채팅 관리, 대회, 위키 도우미, 포럼 관리 |
-| ![][flag_PL] [Yason](https://osu.ppy.sh/users/2574392) | 폴란드어 | 채팅 관리 |
-| ![][flag_CN] [Zero__wind](https://osu.ppy.sh/users/1822830) | 중국어 | 채팅 관리 |
+| ![][flag_CL] [WalterToro](https://osu.ppy.sh/users/5281416) | 스페인어 | 채팅 관리, 대회 관리, wiki 관리, 포럼 관리 |
+| ![][flag_PL] [Yason](https://osu.ppy.sh/users/2574392) | <!-- TODO --> | <!-- TODO --> |
+| ![][flag_HM] [Zallius](https://osu.ppy.sh/users/55) | <!-- TODO --> | 채팅 관리 |
 | ![][flag_DE] [Zetera](https://osu.ppy.sh/users/587737) | 독일어 | 채팅 관리 |
 | ![][flag_TR] [Zeus-](https://osu.ppy.sh/users/5464437) | 터키어 | 채팅 관리 |
 
+### 언어별 모더레이터
```
Comment (Mikeds36) [inline wiki/People/The_Team/Global_Moderation_Team/ko.md]:
```suggestion
### 언어별 관리자
```

Comment (dora-0) [reply wiki/People/The_Team/Global_Moderation_Team/ko.md]:
```suggestion
### 언어 별 관리자
```

File: wiki/People/The_Team/Global_Moderation_Team/ko.md [hunk 3]
```diff
@@ -1,79 +1,118 @@
...
+
+| 채널 | 서브 포럼 | 모더레이터 |
```
Comment (dora-0) [inline wiki/People/The_Team/Global_Moderation_Team/ko.md]:
```suggestion
| 채널 | 서브 포럼 | 관리자 |
```

**Comment by dora-0** [comment]
@Mikeds36 resolved

**Comment by dora-0** [comment]
Ready for merge

File: wiki/People/The_Team/Global_Moderation_Team/ko.md [hunk 4]
```diff
@@ -1,79 +1,118 @@
...
 ## 역할과 책임
 
-Global Moderation Team은 채팅 채널과 포럼의 쾌적한 운용을 책임지고 커뮤니티의 대화에 신경을 써야합니다. 만약 여러분이 채팅(사일, 괴롭힘 등)이나 포럼(스팸)에 대해서 (이것에 한정된 것은 아님) 채팅의 중재 및 게임에 관련되어 도움이 필요하다면 이들에게 연락해주세요.
+Global Moderation Team은 채팅 채널과 포럼의 쾌적한 운용을 책임지고 커뮤니티의 대화에 신경을 써야합니다. 만약 여러분이 채팅(채팅 제한, 괴롭힘 등)이나 포럼(스팸)에 관한 문제(이것에 한정된 것은 아님), 또는 게임과 관련된 도움이 필요하다면 이들에게 연락해주세요.
 
 ## 팀 구성원
 
+[Global Moderation Team 그룹 페이지](https://osu.ppy.sh/groups/4)에서 모든 멤버의 목록을 확인할 수 있습니다.
+
 *참고: 특별한 언급이 없는 한 모든 GMT 구성원들은 영어를 구사할 수 있습니다.* — [팀 페이지](https://osu.ppy.sh/groups/4)
```
Comment (MegaApplePi) [inline wiki/People/The_Team/Global_Moderation_Team/ko.md]:
```suggestion
*참고: 특별한 언급이 없는 한 모든 GMT 구성원들은 영어를 구사할 수 있습니다.*
```

**Pull request merged**